pom.xml

```diff
@@ -0,0 +1,236 @@
+      <organizationUrl>http://github.com/uclalibrary</organizationUrl>
+      <timezone>America/Los_Angeles</timezone>
+    </developer>
+    <developer>
+      <id>ksclarke</id>
+      <name>Kevin S. Clarke</name>
+      <email>ksclarke@ksclarke.io</email>
+      <organization>UCLA Library</organization>
+      <organizationUrl>http://github.com/uclalibrary</organizationUrl>
+      <timezone>America/New_York</timezone>
+    </developer>
+    <developer>
+      <id>cachemeoutside</id>
+      <name>Anthony Vuong</name>
+      <email>avuong@cachemeoutside.io</email>
+      <organization>UCLA Library</organization>
+      <organizationUrl>http://github.com/uclalibrary</organizationUrl>
+      <timezone>America/Los_Angeles</timezone>
+    </developer>
+    <developer>
+      <id>markmatney</id>
+      <name>Mark A. Matney, Jr.</name>
+      <email>mmatney@library.ucla.edu</email>
+      <organization>UCLA Library</organization>
+      <organizationUrl>http://github.com/uclalibrary</organizationUrl>
+      <timezone>America/Los_Angeles</timezone>
+    </developer>
+  </developers>
+
+  <repositories>
+    <repository>
+      <id>mvnrepository.com</id>
+      <url>http://mvnrepository.com</url>
+    </repository>
+    <repository>
+      <id>repo1.maven.org</id>
+      <url>https://repo1.maven.org/maven2/</url>
+    </repository>
+  </repositories>
+
+  <properties>
+    <assembly.version>3.1.1</assembly.version>
+    <codacy.version>1.2.0</codacy.version>
+    <junit.version>4.13</junit.version>
+    <ffmpeg.version>0.6.2</ffmpeg.version>
+    <opencsv.version>4.6</opencsv.version>
+    <picoli.version>4.5.1</picoli.version>
+    <system-lambda.version>1.1.0</system-lambda.version>
+    <system-rules.version>1.19.0</system-rules.version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>net.bramp.ffmpeg</groupId>
+      <artifactId>ffmpeg</artifactId>
+      <version>${ffmpeg.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.opencsv</groupId>
+      <artifactId>opencsv</artifactId>
+      <version>${opencsv.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>info.picocli</groupId>
+      <artifactId>picocli</artifactId>
+      <version>${picoli.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>${junit.version}</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.github.stefanbirkner</groupId>
+      <artifactId>system-lambda</artifactId>
+      <version>${system-lambda.version}</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.github.stefanbirkner</groupId>
+      <artifactId>system-rules</artifactId>
+      <version>${system-rules.version}</version>
+      <scope>test</scope>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <finalName>csvMetaAdder</finalName>
```

Comment:
I'm curious why you want to override the default: `${artifactId}-${version}`? Isn't it useful to have a version number on the Jar?

Reply:
A legacy of my old web app programming; used `finalName` when building WARs, to get a simple name for the final build; didn't use version numbers, because Tomcat uses the WAR name for the app/service context path, and including version would change the path on every deploy, which would be _slightly_ inconvenient for users...

Reply:
Ah, okay... that makes sense.